tests/test_api.py

```diff
@@ -2532,6 +2541,10 @@ def test_simple(self):
         if not self.sg.server_caps.version or self.sg.server_caps.version < (6, 2, 0):
             return
 
+        user_entity = "ApiUser"
+        if self.config.jenkins:
+            user_entity = "HumanUser"
```

Comment:
Minor: you could use a ternary assignment here instead. Would make sense because you did similar above.